2-mandatory/2-currency-code-lookup.js

```diff
@@ -18,9 +18,12 @@ const COUNTRY_CURRENCY_CODES = [
 ];
 
 function createLookup(countryCurrencyCodes) {
-  // write code here
+  let obj = {};
+  for (let item of countryCurrencyCodes) {
+    obj[`${item[0]}`] = item[1]
```

Comment:
This implementation looks correct. But do you need to use string interpolation here? Maybe you can simplify this?